From 34ae55b1cd3f5fb5d2f0bd84bbb81c4341607664 Mon Sep 17 00:00:00 2001
From: vidhum <vidhum@xilinx.com>
Date: Tue, 21 Jun 2011 11:54:04 +0530
Subject: [PATCH 159/628] Xilinx: ARM: Device Config Driver: Completed device
 tree support

git://github.com/Xilinx/linux-xlnx.git xilinx-v14.7
commit f53b50917341137b2feb21e8e2f3f8a9e29d139a

The platform driver structure in driver file is updated with the
.of_match_table parameter.
The devices.c file is updated with the related device-tree changes.

Signed-off-by: Vidhumouli H <vidhum@xilinx.com>
Signed-off-by: Liming Wang <liming.wang@windriver.com>
---
 drivers/char/xilinx_devcfg.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/char/xilinx_devcfg.c b/drivers/char/xilinx_devcfg.c
index 4fec5a5..776014e 100644
--- a/drivers/char/xilinx_devcfg.c
+++ b/drivers/char/xilinx_devcfg.c
@@ -1469,6 +1469,7 @@ static struct platform_driver xdevcfg_platform_driver = {
 	.driver = {
 		.owner = THIS_MODULE,
 		.name = DRIVER_NAME,
+		.of_match_table = xdevcfg_of_match,
 	},
 };
 
-- 
1.7.5.4

